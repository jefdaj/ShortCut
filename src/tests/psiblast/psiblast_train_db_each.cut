query  = load_faa "data/7942gene.faa"
db6803 = makeblastdb_prot_all [load_faa "data/6803genes.faa"]
db7942 = makeblastdb_prot_all [load_faa "data/7942genes.faa"]
# pssm   = psiblast_train_db 1.0e-2 query db7942
pssm   = psiblast_train_db_each 1.0e-2 query [db6803, db7942]
result = pssm
