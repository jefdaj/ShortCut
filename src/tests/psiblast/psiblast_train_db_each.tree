.
|-- cache
|   |-- lines
|   |   |-- 33ec325d48.txt
|   |   |-- 39005f8006.txt
|   |   |-- 40e919f730.txt
|   |   |-- 45121d690b.txt
|   |   |-- 8abd22b6c2.txt
|   |   |-- 902cb58552.txt
|   |   |-- 908f92d67a.txt
|   |   |-- a1512fc008.txt
|   |   |-- ab87e072bf.txt
|   |   |-- adfdb6bd8c.txt
|   |   |-- bdc783547b.txt
|   |   `-- e00fb07ddb.txt
|   |-- load
|   |   |-- 82645ed6f6.faa -> $WORKDIR/data/7942gene.faa
|   |   |-- 8aa4a9d2fc.faa -> $WORKDIR/data/6803genes.faa
|   |   `-- a810086108.faa -> $WORKDIR/data/7942genes.faa
|   |-- makeblastdb
|   |   |-- a1512fc008
|   |   |   |-- a1512fc008.pdb.phr
|   |   |   |-- a1512fc008.pdb.pin
|   |   |   `-- a1512fc008.pdb.psq
|   |   `-- e00fb07ddb
|   |       |-- e00fb07ddb.pdb.phr
|   |       |-- e00fb07ddb.pdb.pin
|   |       `-- e00fb07ddb.pdb.psq
|   |-- map
|   |   |-- 3d6441b9b3.pssm -> ../../cache/lines/39005f8006.txt
|   |   `-- c7f54bfe5e.pssm -> ../../cache/lines/40e919f730.txt
|   `-- psiblast
|-- exprs
|   |-- list
|   |   |-- 10cb52d92f.faa.list -> ../../cache/lines/e00fb07ddb.txt
|   |   |-- 9c1c85aca7.pdb.list -> ../../cache/lines/45121d690b.txt
|   |   `-- d557a7caa8.faa.list -> ../../cache/lines/a1512fc008.txt
|   |-- load_faa
|   |   |-- 33c746245c.faa -> ../../cache/load/82645ed6f6.faa
|   |   |-- 387f499ddc.faa -> ../../cache/load/a810086108.faa
|   |   `-- dee24d6c3c.faa -> ../../cache/load/8aa4a9d2fc.faa
|   |-- makeblastdb_prot
|   |   |-- 27a934c58e.pdb -> ../../cache/lines/908f92d67a.txt
|   |   `-- ced061f0b2.pdb -> ../../cache/lines/902cb58552.txt
|   |-- num
|   |   `-- c23938f494.num -> ../../cache/lines/33ec325d48.txt
|   |-- psiblast_train_db_each
|   |   `-- f51c91829c_4d0e43d616_94686d4973.pssm.list -> ../../cache/lines/ab87e072bf.txt
|   `-- str
|       |-- 1d8a61c770.str -> ../../cache/lines/bdc783547b.txt
|       |-- b9eaf59b0c.str -> ../../cache/lines/8abd22b6c2.txt
|       `-- ec3b181d8a.str -> ../../cache/lines/adfdb6bd8c.txt
`-- vars
    |-- db6803.pdb -> ../exprs/makeblastdb_prot/27a934c58e.pdb
    |-- db7942.pdb -> ../exprs/makeblastdb_prot/ced061f0b2.pdb
    |-- pssm.pssm.list -> ../exprs/psiblast_train_db_each/f51c91829c_4d0e43d616_94686d4973.pssm.list
    |-- query.faa -> ../exprs/load_faa/33c746245c.faa
    `-- result -> ../vars/pssm.pssm.list

16 directories, 41 files
