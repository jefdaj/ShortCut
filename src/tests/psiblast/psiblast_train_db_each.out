[>Synpcc7942_0002 hypothetical protein (trained on a1512fc008.pdb with e-value...
Last position-specific scoring matrix computed, weighted observed percentages...
            A   R   N   D   C   Q   E   G   H   I   L   K   M   F   P   S   T...
    1 M    -1  -1  -2  -3  -1   0  -2  -3  -2   1   2  -1   5   0  -2  -1  -1...
    2 A     4  -1  -2  -2   0  -1  -1   0  -2  -1  -1  -1  -1  -2  -1   1   0...
...
,
 >Synpcc7942_0002 hypothetical protein (trained on e00fb07ddb.pdb with e-value...
Last position-specific scoring matrix computed, weighted observed percentages...
            A   R   N   D   C   Q   E   G   H   I   L   K   M   F   P   S   T...
    1 M    -1  -1  -2  -3  -1   0  -2  -2  -1   1   2  -1   5   0  -2  -1  -1...
    2 A     4  -1  -1  -2   0  -1  -1   0  -2  -1  -1  -1  -1  -2  -1   1   0...
...
]
